frontend/glideinFrontendElement.py

```diff
@@ -244,8 +232,7 @@ def configure(self):
             if not proxy_plugins.get(self.elementDescript.merged_data["ProxySelectionPlugin"]):
                 logSupport.log.warning(
                     "Invalid ProxySelectionPlugin '%s', supported plugins are %s"
-                    % (self.elementDescript.merged_data["ProxySelectionPlugin"]),
-                    list(proxy_plugins.keys()),
+                    % (self.elementDescript.merged_data["ProxySelectionPlugin"], list(proxy_plugins.keys()))
```

Comment:
Better to change this to f-string?

Reply:
Unless you think is very important, I prefer this way here, it allows to split the line

Reply:
As discussed during the standup, the choice was to go with readability in this case since the line is quite long. But generally, consistency is usually preferred.